presentation/src/main/java/com/cointrend/presentation/ui/about/AboutScreen.kt

```diff
@@ -0,0 +1,236 @@
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.modifier.modifierLocalConsumer
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cointrend.presentation.R
+import com.cointrend.presentation.commoncomposables.*
+import com.cointrend.presentation.theme.StocksDarkBackgroundTranslucent
+import com.cointrend.presentation.theme.StocksDarkSecondaryText
+import com.cointrend.presentation.ui.coindetail.SectionInfoItem
+
+
+private val defaultHorizontalPadding = 16.dp
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AboutScreen(
+) {
+    val focusManager = LocalFocusManager.current
+
+    Scaffold(
+        modifier = Modifier,
+        topBar = {
+            TopAppBar(
+                title = {IconButton(onClick = {
+                    System.out.println("Go Back")
+                }) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_arrow_back_ios),
+                        contentDescription = "Return to previous screen",
+                        modifier = Modifier.padding(start = 12.dp)
+                    )
+                }
+                }
+            )
+        }
+    ) { padding ->
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(padding)
+                .pointerInput(Unit) {
+                    detectTapGestures(
+                        onTap = { focusManager.clearFocus() },
+                        onPress = { focusManager.clearFocus() },
+                        onLongPress = { focusManager.clearFocus() },
+                        onDoubleTap = { focusManager.clearFocus() }
+                    )
+                }
+        ) {
+            LazyColumn() {
+                // The Logp
+                item {
+                    CoinTrend()
+                }
+
+                // Setion title 16dp SOURCE CODE
+                item {
+                    SectionTitle(title = "Source Code", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Github", info = "https://github.com/CoinTrend/CoinTrend", image = R.drawable.ic_github, showDivider = false)
+                    }
```

Comment:
In this column let's add also an item
"Changelog" -> "https://github.com/CoinTrend/CoinTrend/releases"